CITATION.cff

```diff
@@ -0,0 +1,43 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Bezanson"
+  given-names: "Jeff"
+- family-names: "Edelman"
+  given-names: "Alan"
+- family-names: "Karpinski"
+  given-names: "Stefan"
+- family-names: "Shah"
+  name-particle: "B"
+  given-names: "Viral"
+title: "Julia: A fresh approach to numerical computing"
+version: 1.8.0-DEV
+license: "MIT"
+doi: "10.1137/141000671"
+date-released: 2017-02-07
+url: "https://github.com/JuliaLang/julia"
+references:
+  - scope: "Cite this paper whenever you use Julia"
+    type: article
+    title: "Julia: A fresh approach to numerical computing"
+    authors:
+      - family-names: "Bezanson"
+        given-names: "Jeff"
+      - family-names: "Edelman"
+        given-names: "Alan"
+      - family-names: "Karpinski"
+        given-names: "Stefan"
+      - family-names: "Shah"
+        name-particle: "B"
+        given-names: "Viral"
+    doi: "10.1137/141000671"
+    journal: "SIAM Review"
+    voume: 59
+    number: 1
+    pages: 13
```

Comment:
According to the [CFF documentation](https://github.com/citation-file-format/citation-file-format/blob/main/README.md#reference-keys) we don't have anything like the `pages={65--98}`:



`pages` | Integer | The number of pages of the work
-- | -- | --

Reply:
Can we put `65-98` in pages? Or will it freak out?

Reply:
I don't know... 😂 could not find a `.cff` parser online and the only thing close to that was the GitHub repository (after adding the `.cff` file)

Reply:
It's odd that everything has to be mentioned twice.

Reply:
> Can we put `65-98` in pages? Or will it freak out?


Here's the schema, I think: https://github.com/citation-file-format/citation-file-format/blob/main/schema.yaml

It says that `pages` is an `int`.

Reply:
I think `start` and `end` are the start and end pages, respectively.